models/User.js

```diff
@@ -123,6 +123,20 @@ const User = sequelize.define('user', {
         allowNull: true,
         defaultValue: ''
     },
+    primary_email: {
+        type: Sequelize.ENUM('personal', 'gsuite'),
+        allowNull: false,
+        defaultValue: 'personal'
+    },
+    notification_email: {
+        type: Sequelize.VIRTUAL,
+        get() {
+            if (this.primary_email === 'gsuite') {
+                return this.gsuite_id;
+            }
+            return this.primary_email;
```

Comment:
This seems wrong. It should return the email right? Not 'personal'

Reply:
Ah yes true, it should be `return this.email`